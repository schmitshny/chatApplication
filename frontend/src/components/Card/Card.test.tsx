import { screen } from '@testing-library/react';
import { render } from '../../tests/test-utils';
import { describe, expect, test } from 'vitest';
import { Card } from '.';

describe('Card Component', () => {
  test('renders with children correctly', () => {
    const testMessage = 'Hello, Card!';
    const { container } = render(
      <Card>
        <p>{testMessage}</p>
      </Card>,
    );

    expect(screen.getByText(testMessage)).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
