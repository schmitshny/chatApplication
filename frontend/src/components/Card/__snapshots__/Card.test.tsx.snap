// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card Component > renders with children correctly 1`] = `
<div>
  <div
    class="sc-bczSft hIodvL"
  >
    <p>
      Hello, Card!
    </p>
  </div>
</div>
`;
